apps/condo/domains/property/schema/Property.test.js

```diff
@@ -125,7 +125,73 @@ describe('Property', () => {
         })
     })
 
-    test('user: cannot update Property address when other Property with same address exist in organization', async () => {
+    test('user: can create Property when same address exist in other organization', async () => {
+        const client = await makeClientWithRegisteredOrganization()
+        const client2 = await makeClientWithProperty()
```

Comment:
You don't need to create client with property, because you are not using it in tests
```suggestion
        const client2 = await makeClientWithRegisteredOrganization()
```